documentation/docs/guides/tool-permissions.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Managing Tool Permissions
+sidebar_position: 4
```

Comment:
@JohnMAustin78 i dont remember your system for sidebar positions. please forgive me and feel free to update. I just put it close to the other permissions guide